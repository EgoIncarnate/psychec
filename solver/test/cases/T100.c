void f()
{
    T1 a;
    a.x = 10;

    T2 b;
    b.x = 9.9;
    b.x = c;
}
