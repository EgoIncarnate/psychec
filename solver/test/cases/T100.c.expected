typedef  struct __T2__ {double x;} T2 ;
typedef  struct __T1__ {int x;} T1 ;
double c ;
